v1/src/simulator/src/circuit.ts

```
@@ -171,113 +188,108 @@ export function getDependenciesList(scopeId: string | number) {
+    if (id) scope.id = id
+    scopeList[scope.id] = scope
+    let currCircuit = {
+        id: scope.id,
+        name: scope.name, // fix for tab name issue - vue - to be reviewed @devartstar
+    }
+
+    circuit_list.value.push(currCircuit)
+    if (isVerilog) {
+        scope.verilogMetadata.isVerilogCircuit = true
+        // TODO: remove later if not required after fixing verilog code loading from saved file
+        circuit_list.value.forEach((circuit) => (circuit.isVerilog = false))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)